crux-core/src/crux/api.clj

```diff
@@ -104,18 +104,10 @@
   (status [node]
     "Returns the status of this node as a map.")
 
-  (submitted-tx-updated-entity? [node submitted-tx eid]
-    "Checks if a submitted tx did update an entity.
-  submitted-tx must be a map returned from `submit-tx`
-  eid is an object that can be coerced into an entity id.
-  Returns true if the entity was updated in this transaction.")
-
-  (submitted-tx-corrected-entity? [node submitted-tx ^Date valid-time eid]
-    "Checks if a submitted tx did correct an entity as of valid time.
-  submitted-tx must be a map returned from `submit-tx`
-  valid-time valid time of the correction to check.
-  eid is an object that can be coerced into an entity id.
-  Returns true if the entity was updated in this transaction.")
+  (tx-committed? [node submitted-tx]
+    "Checks if a submitted tx was successfully committed.
+     submitted-tx must be a map returned from `submit-tx`.
+     Returns true if the submitted transaction was committed.")
```

Comment:
I'd specify the potential to throw `NodeOutOfSyncException` here - as a developer reading the docstring, I'd want to know that this was a case I should handle.

I might also be tempted to explicitly specify why this function might return false, just to distinguish it from the exception case?